aws-amplify/amplify-flutter
# chore: pin Amplify iOS to '~> 1.9.2'

*Issue #, if available:* n/a

*Description of changes:*
- set all Amplify iOS libraries to `'~> 1.9.2'`

This will allow for any version of amplify 1.9.X greater than or equal to 1.9.2. At the time the PR was opened, 1.9.2 was the latest stable release of Amplify iOS.

### Testing locally
**Testing with Amplify iOS 1.9.2:**
- Pull in the changes from the PR (or just make them locally)
- Run `pod update Amplify AWSPluginsCore AmplifyPlugins` from the iOS dir in any sample project you want to test in. See docs PR (https://github.com/aws-amplify/docs/pull/3251) for more info.
- Confirm in the `Podfile.lock` for the project that you have any version 1.9.X equal to or greater than 1.9.2.
- Build example project and test standard functionality

**Testing the upgrade experience:**
- Pull in the changes from the PR (or just make them locally).
- Change `'~> 1.9.2'` to anything earlier than 1.9.X, for example: `'~> 1.6.0'`
- Run `pod update Amplify AWSPluginsCore AmplifyPlugins` from the iOS dir in any sample project you want to test in.
- Confirm in the `Podfile.lock` shows the older version.
- Build the example app, see that it builds/runs, stop the example app
- Change the version back to `'~> 1.9.2'`.
- Attempt to build the app for iOS. 
- Observe the error `CocoaPods could not find compatible versions for pod "Amplify"`.
- Run `pod update Amplify AWSPluginsCore AmplifyPlugins` from the iOS dir
- Observe that the `Podfile.lock` is updated
- Attempt to build the app and observe that it builds/runs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
